manifests/nodejs.yml

```diff
@@ -1032,7 +1033,7 @@ tests/:
     test_shell_execution.py:
       Test_ShellExecution: *ref_5_3_0
     test_suspicious_attacker_blocking.py:
-      Test_Suspicious_Attacker_Blocking: missing_feature
+      Test_Suspicious_Attacker_Blocking: *ref_5_58_0
```

Comment:
59.0.0 no ?

Reply:
Suspicious Attacker Blocking, which relies in exclusion data, works from v5.58.0 onwards. The reason is the new waf update thing, implemented in this [PR](https://github.com/DataDog/dd-trace-js/pull/5792)

Reply:
You are right